.github/workflows/deploy-docker-image.yml

```diff
@@ -133,5 +77,104 @@ jobs:
       actions: read
     secrets: inherit
     with:
-      image: ${{ needs.deploy-docker-image.outputs.image }}
+      image: ${{ needs.build-docker-image.outputs.image }}
       team: k9saksbehandling
+
+  deploy-dev:
+    name: Deploy to dev
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    env:
+      CLUSTER: dev-fss
```

Comment:
vi kan vel denne env-variabelen og bare skrive dev-fss og prod-fss rett i RESOURCE

Reply:
Ja, einaste er at den er brukt to plasser, både i CLUSTER og RESOURCE. Så må hardkode den inn begge stader då. Men kan godt gjere det viss du synest det er betre.